elcid/models.py

```diff
@@ -75,6 +75,12 @@ def age(self):
 
     class Meta:
         verbose_name_plural = "Demographics"
+        constraints = [
+            models.CheckConstraint(
+                check = ~models.Q(hospital_number__startswith=0),
```

Comment:
I think I landed at either a different spelling or a comment explaining how _bitwise_ not works here...